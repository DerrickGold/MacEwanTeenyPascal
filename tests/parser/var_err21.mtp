(* variable section start *)

(*variable listing, using keyword as an id*)
var test, integer : integer;

begin

end
.