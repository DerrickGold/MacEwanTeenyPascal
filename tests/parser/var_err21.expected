Syntax Error: line 4:
Unexpected token (TOK_KEY_INTEGER, 'integer'), likely missing a(n)  
  'TOK_SEMICOLON'
